banjo/builder.cpp

```diff
@@ -835,6 +835,12 @@ Builder::make_aggregate_init(Type& t, Expr_list const& es)
 // -------------------------------------------------------------------------- //
 // Declarations
 
+Super_decl&
+Builder::make_super_declaration(Name& n, Type& t)
```

Comment:
I wonder if we can synthesize the name here too. This should really only take the type as an argument.